<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(219, 236, 255)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QWidget" name="widget_14" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">
font: 700 italic 25pt &quot;Georgia&quot;;
</string>
        </property>
        <property name="text">
         <string>Verify it's You</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_15" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_11" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_17" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QWidget" name="widget_10" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>12</y>
              <width>33</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="ea_li">
            <property name="geometry">
             <rect>
              <x>53</x>
              <y>12</y>
              <width>125</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white
</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_12" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_18" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QWidget" name="widget_19" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>12</x>
              <y>12</y>
              <width>60</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="password_li">
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>12</y>
              <width>125</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background: white
</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_20" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget_5" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="Med1">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(190, 213, 255)</string>
        </property>
        <property name="text">
         <string>Med1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="Med2">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(190, 213, 255)</string>
        </property>
        <property name="text">
         <string>Med2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_16" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="Med3">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(190, 213, 255)</string>
        </property>
        <property name="text">
         <string>Med3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="backtoMain2">
        <property name="styleSheet">
         <string notr="true">background:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Main Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
